addition_1.txt

```diff
@@ -1 +1 @@
-This is a file added to a new branch one for test purposes
+This is a file added to a new branch one for test purposes.
```

Comment:
So I can add yet another comment to that? Makes sense.
I could add the comment to the initial commit or so it looks. Didn't have to add dot after all.

Reply:
Pretty much. Let me check if there are more places I can add comments, and add it to the initial commit, too.

Reply:
Delete the latest commit please